### PR TITLE
Create new issue template

### LINKED ISSUE #19975: Provide issue/pr github template
We can include a bunch of "did you check X" etc in these templates, see here: https://github.com/blog/2111-issue-and-pull-request-templates

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #19975 

What do you think about this wording?

### CODE REVIEW

**Comment by johanandren** [comment]
This is more like something we'd like to show above the input field when creating an issue than a template helping with what to fill in?

**Comment by ktoso** [comment]
> This is more like something we'd like to show above the input field when creating an issue than a template helping with what to fill in?

There's no way to do that though

**Comment by johanandren** [comment]
Ah, I thought the yellow box was ours, but I see now it only links to contributing and the code of conduct.

I'm on the fence about this, I can see how adding the information would be useful, but annoying to always have to start with removing a chunk of text when opening an issue.

**Comment by ktoso** [comment]
That's the thing, you don't have to remove it since it's a comment

**Comment by johanandren** [comment]
Aah, clever, didn't understand that!

**Review by johanandren — APPROVED** [review]

**Comment by patriknw** [comment]
yeah, I had the the same arguments against this originally, but with these comments I think ti can work.

**Comment by ktoso** [comment]
Let's try then. I'm not sure if the text will appear in emails though... I hope not.
If it's a hassle we can remove it

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- 
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka-HTTP: https://github.com/akka/akka-http/issues 
+ - Alpakka:   https://github.com/akka/alpakka/issues 
+ - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - ...
+
```
Comment (patriknw) [inline .github/ISSUE_TEMPLATE.md]: Let's add one sentence about that questions are not welcome here.

Comment (ktoso) [reply .github/ISSUE_TEMPLATE.md]: Ok adding

**Pull request merged**